ssv/spectest/tests/runner/consensus/post_finish.go

```diff
@@ -14,7 +16,7 @@ import (
 func PostFinish() tests.SpecTest {
 	ks := testingutils.Testing4SharesSet()
 
-	return &tests.MultiMsgProcessingSpecTest{
+	multiSpecTest := &tests.MultiMsgProcessingSpecTest{
```

Comment:
the test doesn't do what it says.. it suppose to process a commit msg after runner finished but in the test and state comparison it doesn't finish. 
We need to be attentive about state comparison and understand the actual test!